yal/askaquestion.py

```diff
@@ -52,12 +52,11 @@ async def state_aaq_start(self):
         question = self._(
             "\n".join(
                 [
-                    "🙋🏿‍♂️ *QUESTIONS?*",
-                    "Ask A Question",
+                    "🙋🏿‍♂️ QUESTIONS? / *Ask A Question*",
                     "-----",
                     "",
-                    "🙍🏾‍♀️*That's what I'm here for!*",
-                    "*Just type your Q and hit send* .🙂.",
+                    "[persona_emoji] *That's what I'm here for!*",
```

Comment:
Should this be a to do? I don't see this being replaced with the emoji anywhere

Reply:
Nope. this is the placeholder we're using. my next ticket is to replace the placeholder with an emoji across all messages.